### PR TITLE
Run release tests on new stack

### PR DESCRIPTION
**Pull request by shomilj**
## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/nightly_tests/chaos_test/compute_template.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cloud_id: cld_4F7k8814aZzGG8TNUGPKnc
+cloud_id: cld_kvedZWag2qA8i5BjxUevf5i7
```
Comment (Yard1) [inline release/nightly_tests/chaos_test/compute_template.yaml:1]:
```suggestion
cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
```

File: release/dashboard/mem_check.py [hunk 2]
```diff
@@ -15,6 +15,8 @@
     client = JobSubmissionClient("http://127.0.0.1:8265")
     job_id = client.submit_job(
```
Comment (Yard1) [inline release/dashboard/mem_check.py:16]: I have edited this test in my anyscale jobs PR to allow it to have a configurable working dir. Feel free to copy https://github.com/ray-project/ray/blob/2aff537a924a6f5befeaf6f477867a2c3dc31854/release/dashboard/mem_check.py

File: release/ray_release/cluster_manager/full.py [hunk 3]
```diff
@@ -91,6 +91,11 @@ def start_cluster(self, timeout: float = 600.0):
             )
             completed = result.result.completed
 
+        # Sleep 30 seconds to let everything come up (this is a short-term hack to see if
+        # we can get release tests working on the new stack, since it seems like session
+        # state operations are not indicative of the cluster being externally available).
+        time.sleep(30)
```
Comment (Yard1) [inline release/ray_release/cluster_manager/full.py:97]: should not be necessary as we have wait for nodes - it just needs a shorter timeout

File: release/ray_release/environments/prod.env [hunk 4]
```diff
@@ -1,5 +1,5 @@
-ANYSCALE_HOST=https://console.anyscale.com
-RELEASE_AWS_ANYSCALE_SECRET_ARN="arn:aws:secretsmanager:us-west-2:029272617770:secret:release-automation/anyscale-token20210505220406333800000001-BcUuKB"
-RELEASE_DEFAULT_CLOUD_ID="cld_4F7k8814aZzGG8TNUGPKnc"
-RELEASE_DEFAULT_PROJECT="prj_FKRmeV5pA6X72aVscFALNC32"
-ANYSCALE_PROJECT="prj_FKRmeV5pA6X72aVscFALNC32"
\ No newline at end of file
+ANYSCALE_HOST=https://console.anyscale-staging.com
```
Comment (Yard1) [inline release/ray_release/environments/prod.env:1]: I am not sure we should be turning the prod env into copy of staging

**Review by Yard1 — COMMENTED** [review]
thanks, looks great! left a few comments so far

File: release/ray_release/environments/prod.env (hunk 4 above)
Comment (shomilj) [reply release/ray_release/environments/prod.env:1]: True, it's a hack, I will make a new PR to do this correctly

**Pull request closed without merging**